client/components/data/domain-management/email-forwarding/index.jsx

```diff
@@ -19,26 +20,26 @@ function getStateFromStores( props ) {
 	};
 }
 
-const EmailForwardingData = React.createClass( {
-	propTypes: {
-		component: React.PropTypes.func.isRequired,
-		selectedDomainName: React.PropTypes.string.isRequired,
-		sites: React.PropTypes.object.isRequired
-	},
+export class EmailForwardingData extends Component {
+	static propTypes = {
+		component: PropTypes.func.isRequired,
+		selectedDomainName: PropTypes.string.isRequired,
+		selectedSite: PropTypes.object,
+	};
 
-	mixins: [ observe( 'sites' ) ],
+	constructor( props ) {
+		super( props );
 
-	componentWillMount() {
 		this.loadEmailForwarding();
```

Comment:
For anyone reading:
```
Generally, we recommend using the constructor() instead.
```
https://facebook.github.io/react/docs/react-component.html#componentwillmount